### ISSUE TITLE
a11y: fieldset and legend usage for form elements

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#781]
**What Should Happen**
`<fieldset>` and `<legend>` should be used to group and describe related form elements, and should not be used for other reasons.

**JIRA Issues**
- [x] MOVE-129
- [x] MOVE-106
- [x] MOVE-41
- [x] MOVE-40
- [x] MOVE-23

**Comment by candu** [comment]
Punted on a few tasks related to the use of `<fieldset>` and `<legend>` as styling elements in Vuetify inputs with the `outlined` prop - changing this would be a lot of either design work (if we want to change to another form element style, such as the default "underlined" style, throughout our forms) or dev work (if we want to mimic the existing `outlined` style without `<fieldset>`, which would basically require us to build these components from the ground up.)
